settings.gradle

```diff
@@ -4,8 +4,8 @@ include ':app'
 // We assume, that NewPipe and NewPipe Extractor have the same parent directory.
 // If this is not the case, please change the path in includeBuild().
 
-//includeBuild('../NewPipeExtractor') {
-//    dependencySubstitution {
-//        substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
-//    }
-//}
+includeBuild('../NewPipeExtractor') {
+    dependencySubstitution {
+        substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
+    }
+}
```

Comment:
Please undo the changes in this file, they are not meant to be committed! ;-)
Instead you should change the `NewPipeExtractor` line in `app/build.gradle` (699b4086398f1e0f412e178532e85c7a23fadf83 is the last commit in your extractor PR):
```
    implementation 'com.github.CaptSzat:NewPipeExtractor:699b4086398f1e0f412e178532e85c7a23fadf83'
```